.travis.yml

```diff
@@ -7,7 +7,7 @@ rvm:
   - 1.9
   - 2.0
   - 2.1
-  - 2.2
+  - 2.2.2
```

Comment:
It's an RVM thing, I believe.. see [here](https://github.com/rvm/rvm/blob/stable/config/known#L10). Specifying just `2.2` resolves to `2.2.1` (I was apparently mistaken above when I said `2.2.0`) in the latest stable release of RVM. (It seems the revision number will jump on the next RVM release as this is different in RVM's `master` branch.)

In any case, I assure you it's necessary. I only changed it after it didn't work.